airflow/_vendor/README.md

```diff
@@ -0,0 +1,37 @@
+# Vendor package
+
+## What vendored packages are for
+
+The `airflow._vendor` package is foreseen for vendoring in packages, that we have to modify ourselves
+because authors of the packages do not have time to modify them themselves. This is often temporary
+and once the packages implement fixes that we need, and then we remove the packages from
+the `_vendor` package.
+
+All Vendored libraries must follow these rules:
+
+1. Vendored libraries must be pure Python--no compiling (so that we do not have to release multi-platform airflow packages on PyPI).
+2. Source code for the libary is included in this directory.
+3. License must be included in this repo and in the [LICENSE](../../LICENSE) file and in the
+   [licenses](../../licenses) folder.
+4. Requirements of the library become requirements of airflow core.
+5. Version of the library should be included in the [vendor.md](vendor.md) file.
+6. No modifications to the library may be made in the initial commit.
```

Comment:
```suggestion
6. No modifications to the library may be made in the initial commit (except import paths).
```

Reply:
The import paths should be separate commit -> this way it is easier to cherry pick changes later. This is what I've done to show it: 

* https://github.com/apache/airflow/pull/22206 -> adding cgroups
* Fix imports: https://github.com/apache/airflow/pull/22207

This will allow to replace code with the next version (equivalent of #2206) and then apply the import change cherry-pick (#22207) in case we need ot upgrade.